screengrab/lib/screengrab/page.html.erb

```diff
@@ -64,117 +64,99 @@
       }
     </style>
   </head>
-  <body>
-    <% divide_size_by = 5.0 %>
-    <% max_width = 180 %>
-    <% image_counter = 0 %>
+  <body><% image_counter = 0 %><% @data.each do |language, content| %>
+    <h1 class="language"><%= language %></h1>
+    <hr>
+    <table><% content.each do |device_name, screens| %>
+      <tr>
+        <th colspan="<%= screens.count %>">
+          <span class="deviceName"><%= device_name %></span>
+        </th>
+      </tr>
+      <tr><% screens.each do |screen_path| %><% next if screen_path.include?"_framed.png" %>
+        <td><% image_counter += 1 %>
+          <a href="<%= screen_path %>" target="_blank" class="screenshotLink">
+            <img class="screenshot" src="<%= screen_path %>" style="width: 100%;" alt="<%= language %> <%= device_name %>" data-counter="<%= image_counter %>">
```

Comment:
Inlined this entire image declaration into one line and also inlined the `style` variable.